docs/source/run/install.rst

```diff
@@ -0,0 +1,144 @@
+=============
+Install Guide
+=============
+
+Requirements
+============
+
+To install Sopel, you will need:
+
+* Python 3.8 or above
+* Pip, the official Python package installer
+
+.. important::
+
+    Before installing Sopel, you should know which version of Python, and which
+    Python interpreter you are using for that. For instance, it can be named
+    ``python3`` or just ``python`` depending on your system and setup. In this
+    guide, we will refer as ``python3`` the Python interpreter you want to use.
+
+Official Release
+================
+
+The official release is available at `pypi.org/project/sopel/`__. The
+recommended way to install Sopel from pypi.org is to perform the following
+command::
+
+    python3 -m pip install sopel
```

Comment:
The default syntax highlighting should be changed to `shell` for these files with a lot of shell command examples. They are highlighted as Python right now, and it looks a bit weird.